extensions/wikia/CommunityPage/CommunityPage.i18n.php

```diff
@@ -31,17 +31,17 @@
 	'communitypage-contributions' => '$1 {{PLURAL:$1|contribution|contributions}}',
 	'communitypage-admins' => 'Admins',
 	'communitypage-other-admins' => 'Other Admins',
-	'communitypage-recently-joined' => 'Recently Joined',
+	'communitypage-recently-joined' => 'Recent New Contributors',
 	'communitypage-recent-activity' => 'View All Recent Activity »',
 	'communitypage-all-members' => 'All Members',
-	'communitypage-view-all-members' => 'View All Members »',
+	'communitypage-view-all-members' => 'View all Contributors »',
```

Comment:
Making "all" lowercase seems inconsistent with the other messages (i.e. "View All Recent Activity"). Is this capitalisation change intentional? And should we change the others?